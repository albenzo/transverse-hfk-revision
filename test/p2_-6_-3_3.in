-i 11 -X [11,10,4,5,2,6,8,7,3,9,1] -O [6,5,7,3,4,9,11,10,1,2,8]