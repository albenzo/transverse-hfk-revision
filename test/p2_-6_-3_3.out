LL is NOT null-homologous
UR is NOT null-homologous
D1[LL] is null-homologous
D1[UR] is NOT null-homologous
